### DISCUSSION TITLE
How to add a header/background image?

### DISCUSSION THREAD

**Discussion by Noeloikeau** [alshedivat/al-folio#1311] [Q&A] [answered]
Hello, thank you for the fantastic site, my question is can I add a background image to the about page (or any/all pages)? For example, https://reactresume.com/ has a simple background rather than the plain white/black of this template, and I would like to implement something similar if possible.

I'm new to html and websites in general but my best guess is editing about.html to include something like 
body 
div style="background-image: url('https://www.website.com/images/image_background.jpg');"
/body
(with angular brackets)
Is this along the right path? And what other modifications might be necessary for consistency?

Thank you very much,
-Noelo

**Comment by george-gca** [comment, marked as answer]
In the top of `about.html` there is a front matter with `layout: default`, which means that this file inherits some content from `default.html`. More exactly, it will replace the `{{ content }}` part inside of it. So if you want to play with the `<body>` tag, there is where it actually is. But if you  want to mess with its css classes, [_sass/_layout.scss](https://github.com/alshedivat/al-folio/blob/master/_sass/_layout.scss) is the file to go.
